presto-docs/src/main/sphinx/connector/hive-security.rst

```diff
@@ -131,6 +131,8 @@ Property Name                                      Description
                                                    to the Hive metastore service.
 
 ``hive.metastore.client.keytab``                   Hive metastore client keytab location.
+``hive.metastore.impersonation.enabled``           Enable metastore end-user impersonation.
+``hive.metastore.impersonation.user``              Default impersonation user when communicating with Hive Metastore
```

Comment:
Why this parameter needed?

Reply:
This is because the getPartitionsStatistics calling from the initializer and need a "default" impersonation user to call, we can make this user as authenticated user in presto process, should be "presto"